pkgs/development/python-modules/ffcv/default.nix

```diff
@@ -24,16 +34,15 @@ buildPythonPackage rec {
     hash = "sha256-L2mwGFivq/gtAw+1D6U2jbW6VxYgetHX7OUrjwyybqE=";
   };
 
+  build-system = [ setuptools ];
```

Comment:
Sorry I didn't mention this in advance: `build-system` (`dependencies`) go somewhere near `nativeBuildInputs` (`buildInputs`), and `postPatch` goes right after `src` and `patches`. It's formatting shtick...

Reply:
Done! Thanks for the feedback.